### PR TITLE
[Crowdstrike] Update Test Log For Alert Data Stream

### PR DESCRIPTION
**Pull request by mohitjha-elastic**
Type of change
- Enhancement

## What does this PR do?
**Update the test log for the alert data stream.**
The intent for this PR is to remove some sensitive data such as username and email from the alert test log. Hence updated the sample test data.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### All changes
- [x] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [x] Supported versions of the monitoring target are documented
- [x] Supported operating systems are documented (if applicable)
- [x] Integration or [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) exist
- [x] Documentation exists
- [x] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [x] At least a manual test with ES / Kibana / Agent has been performed.
- [x] Required Kibana version set to: ^8.11.0

## How to test this PR locally
Clone integrations repo.
Install the elastic package locally.
Start the elastic stack using the elastic package.
Move to integrations/packages/crowdstrike directory.
Run the following command to run tests.
`elastic-package test -v`

### CODE REVIEW

**Review by efd6 — COMMENTED** [review]
Please update the proposed commit message to include an explanation of why this is being done.

File: packages/crowdstrike/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.42.3"
+  changes:
+    - description: Update the test log for alert data stream.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/11390
```
Comment (kcreddy) [inline packages/crowdstrike/changelog.yml:6]: Remove this change log entry altogether.
This is no bugfix or enhancement. Its only the test files that change.

File: packages/crowdstrike/manifest.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: crowdstrike
 title: CrowdStrike
-version: "1.42.2"
+version: "1.42.3"
```
Comment (kcreddy) [inline packages/crowdstrike/manifest.yml:3]:
```suggestion
version: "1.42.2"
```

No need for updating version. Its tests-only change.

**Review by kcreddy — APPROVED** [review]

**Comment by efd6** [comment]
This really should have had a (possibly private) issue associated with it.

**Pull request merged**